extensions/wikia/Qualaroo/scripts/Qualaroo.js

```diff
@@ -70,5 +80,17 @@
 		createCookie('qualaroo_survey_submission');
 	});
 
+	if(ads.context.opts.sourcePointDetection) {
+		document.addEventListener('sp.blocking', function () {
```

Comment:
Is it possible that this event handler will be registered after event was fired? Can we mitigate that problem?